requirements/prod.txt

```diff
@@ -28,8 +27,9 @@ requests
 rich
 scalecodec==1.2.7  # scalecodec should not be changed unless first verifying compatibility with the subtensor's monkeypatching of scalecodec.RuntimeConfiguration.get_decoder_class
 shtab==1.6.5
+starlette~=0.37.2
 substrate-interface==1.7.5
 termcolor
 tqdm
-uvicorn==0.22.0
+uvicorn[standard]==0.29.0  # standard adds uvloop
```

Comment:
this will break code of people that do require nest_asyncio

bittensor as a library should not mandate use of any particular aioloop

This could be put into extra requirement group as opt-in

Reply:
This is still a WIP with a while to go, and I have noted your initial comments regarding this.